package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-fetch",
-  "version": "1.6.1",
+  "version": "1.6.2",
```

Comment:
Could you kindly revert this change, we actually bump version using `npm version` right before publish.